android/src/main/java/com/applovin/reactnative/AppLovinMAXModule.java

```diff
@@ -380,6 +388,20 @@ public void setTestDeviceAdvertisingIds(final ReadableArray rawAdvertisingIds)
         }
     }
 
+    @ReactMethod()
```

Comment:
Can you try to follow the _exact_ ordering we have in our other plugins? Like we have `setCreativeDebuggerEnabled(..)` before `setTestDeviceAdvertisingIds(..)`. Once we start devitating a little we're going to lose track of changes with this file which is used amongst all our plugins.